appendix-petsciicodes.tex

```diff
@@ -5,16 +5,17 @@ \section{PETSCII Codes and CHR\$}
 
 \label{appendix:asciicodes}
 
-You can use the \screentext{PRINT CHR\$(X)} statement to print a character.
-Below is the full table of PETSCII codes you can print by index.  For example, by
+In BASIC,  \screentext{PRINT CHR\$(X)} can be used to print a character from a PETSCII code.
+Below is the full table of the PETSCII codes you can print by index.  For example, by

```

Comment:
Maybe just a personal thing, but it just feels more familiar to me to read it without 'the' in front of PETSCII codes.